templates/checkout.php

```diff
@@ -25,11 +25,18 @@
 	$timestamp = time();
 
 	$context_params = array(
-		'plugin_id'         => $fs->get_id(),
-		'plugin_public_key' => $fs->get_public_key(),
-		'plugin_version'    => $fs->get_plugin_version(),
+		'plugin_id'      => $fs->get_id(),
+		'plugin_version' => $fs->get_plugin_version(),
+        'public_key'     => $fs->get_public_key(),
+        'mode'           => 'dashboard',
+        'plan_id'        => $_GET['plan_id'],
```

Comment:
@fajardoleo using `$_GET` params as is without any validation isn't a good practice and might be insecure in some cases, please verify the param type with `FS_Plugin_Plan::is_valid_id()` and if it isn't, we should probably NOT add that param to the `$context_params` and let the checkout to auto choose the default plan (the one with `is_featured` `true`).